macros/ArticlesByTag.ejs

```diff
@@ -66,9 +66,9 @@ URL_ROOTS.forEach(function (root) {
 
         if (match) {
             ARTICLES.push({
-                title   : p.title,
+                title   : kuma.htmlEscape(p.title),
```

Comment:
This is wrong. Instead, when you print the title (line 89), you should use ``<%=``, which escapes the result for safe output, rather than ``<%-``, which retains the HTML.